scripts/mbedtls_framework/c_wrapper_generator.py

```diff
@@ -25,12 +25,15 @@ def c_declare(prefix: str, name: str, suffix: str) -> str:
     return prefix + name + suffix
 
 
-WrapperInfo = typing.NamedTuple('WrapperInfo', [
+WrapperInfo = NamedTuple('WrapperInfo', [
     ('argument_names', List[str]),
     ('guard', Optional[str]),
     ('wrapper_name', str),
 ])
 
+def blockstr_format(in_str: str, new_lines: int = 1) -> str:
+    """Return a whitespace stripped str, appended with user defined empty lines."""
```

Comment:
The description omits the important aspect of this function, which is that it removes indentation! Please change the one-line description to focus on the indentation removal. In the long description, mention stripping leading and trailing empty lines, the effect of `new_lines`, and a warning that this function is only suitable if the result is supposed to be vertically aligned to column 0.